worlds/jakanddaxter/docs/setup_en.md

```diff
@@ -0,0 +1,169 @@
+# Jak And Daxter (ArchipelaGOAL) Setup Guide
+
+## Required Software
+
+- A legally purchased copy of *Jak And Daxter: The Precursor Legacy.*
+- [The OpenGOAL Mod Launcher](https://jakmods.dev/)
+- [The Jak and Daxter .APWORLD package](https://github.com/ArchipelaGOAL/Archipelago/releases)
+
+At this time, this method of setup works on Windows only, but Linux support is a strong likelihood in the near future. 
+(OpenGOAL itself supports Linux, and the mod launcher is runnable with Python.)
+
+## Preparations
+
+- Dump your copy of the game as an ISO file to your PC.
+- Install the Mod Launcher.
+- If you are prompted by the Mod Launcher at any time during setup, provide the path to your ISO file.
+
+## Installation
+
+***OpenGOAL Mod Launcher***
+
+- Run the Mod Launcher and click `ArchipelaGOAL` in the mod list.
+- Click `Install` and wait for it to complete.
+  - If you have yet to be prompted for the ISO, click `Re-Extract` and provide the path to your ISO file.
+- Click `Recompile`. This may take between 30-60 seconds. It should run to 100% completion. If it does not, see the Troubleshooting section.
+- Click `View Folder`. 
+  - In the new file explorer window, take note of the current path. It should contain `gk.exe` and `goalc.exe`.
+- Verify that the mod launcher copied the extracted ISO files to the mod directory:
+  - `%appdata%/OpenGOAL-Mods/archipelagoal/iso_data` should have *all* the same files as
+  - `%appdata%/OpenGOAL-Mods/_iso_data`, if it doesn't, copy those files over manually.
+  - And then `Recompile` if you needed to copy the files over.
+- **DO NOT PLAY AN ARCHIPELAGO GAME THROUGH THE MOD LAUNCHER.** It will run in retail mode, which is incompatible with Archipelago. We need it to run in debug mode (see below).
+
+***Archipelago Launcher***
```

Comment:
This may end up being a moot point as the OpenGOAL devs transition mods from their unofficial mod launcher to their official game launcher... Another research item. And I have no problem saying this PR ought not to be merged until that happens.

Reply:
Oh... do you mean the `host.yaml` instructions? I ended up following Factorio's example on that. As mentioned earlier, the game's executables may be changing location in the future so I needed to account for that.

Reply:
I think this part still confuses me a bit - to me, these instructions just look like "how to use the archipelago launcher".  Is the impetus here that we just don't need this anymore? Because I imagine most other games still generate seeds, set up yamls, launch clients through the launcher in this same way.